colaraz/lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -138,7 +138,7 @@
       % endif
 
       % if cert_status['status'] == 'downloadable' and enrollment.mode == 'verified' and cert_status['mode'] == 'honor':
-      <!-- [COLARAZ_CUSTOM] Changing ID verification message for the user who has got certificate in honor mode. -->
+        ## [COLARAZ_CUSTOM] Changing ID verification message for the user who has got certificate in honor mode.
```

Comment:
Why do we need to change the comment to ##? As we can see you are using the same commenting syntax in your other file.